UnityProject/Assets/Scripts/Items/Implants/Organs/Ears.cs

```diff
@@ -95,27 +96,24 @@ private void CheckPressure()
 				UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, CheckPressure);
 			}
 
-			var Pressure =
-				RelatedPart.HealthMaster.RegisterTile.Matrix.GetMetaDataNode(RelatedPart.HealthMaster.RegisterTile
-					.LocalPosition).GasMix.Pressure;
+			var localPosition = Matrix.GetLocalPositionFromWorldPosition(Pickupable.UniversalObjectPhysics);
```

Comment:
Pickupable.UniversalObjectPhysics.official position.to local

Reply:
this is much more verbose and clear

Reply:
But you're not passing in a world position Though

Reply:
uhh, no? I'm not.

Reply:
the problem is this is still linked to the matrix the items on, so it would be better if it got the  matrix at the position and then did the conversion to local